Tasks/Gradle/gradletask.ts

```diff
@@ -64,7 +66,7 @@ if (isCodeCoverageOpted && inputTasks.indexOf('clean') == -1) {
     gb.arg('clean'); //if user opts for code coverage, we append clean functionality to make sure any uninstrumented class files are removed
 }
 
-gb.argString(tl.getInput('options', false));
```

Comment:
I'm not sure which is correct for your scenario.

An example: if you call `t.arg('this is one arg'); t.arg('arg2');` then I believe it ultimately gets passed to the child process as `foo.exe "this is one arg" arg2` (node adds quotes added to retain the first arg as a single arg)

Another example: if you call `t.line('these are separate args');` then I believe it utlimately gets passed to the child process as `foo.exe these are separate args`